ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -590,18 +588,7 @@ export default class ConfirmTransactionBase extends Component {
         this._removeBeforeUnload();
 
         sendTransaction(txData)
-          .then(() => {
-            clearConfirmTransaction();
-            this.setState(
-              {
-                submitting: false,
-              },
-              () => {
-                history.push(mostRecentOverviewPage);
```

Comment:
Without this does the user still return to their mostRecentOverviewPage when done confirming tx?

Reply:
Also do we not need to set submitting to false if there is no error?

Reply:
@brad-decker the user was still being returned yes, I can double check though

@adonesky1 issue here was at this point in the flow, the component is unmounted so state changes become moot. (If there's no error redirection happens) I'll confirm though that there isn't a latent race condition issue that I wasn't seeing